src/utils/index.js

```diff
@@ -0,0 +1,25 @@
+export const headerZUID = (response) => {
+  return response?.headers?.get('z-zuid') || '';
+};
+
+export function getCookie(cname) {
```

Comment:
there is another cookie method already being used, see https://github.com/zesty-io/nextjs-website/blob/368e48e22dfaa8567ae2b6582873ad803a10b262/src/components/cta/StandardFormWithSelect.js#L25 import and usage https://github.com/zesty-io/nextjs-website/blob/368e48e22dfaa8567ae2b6582873ad803a10b262/src/components/cta/StandardFormWithSelect.js#L67

remove this class in favor of using the package